test/e2e/system_connection_test.go

```diff
@@ -281,6 +281,20 @@ var _ = Describe("podman system connection", func() {
 			_, err = Start(cmd, GinkgoWriter, GinkgoWriter)
 			Expect(err).ToNot(HaveOccurred())
 
+			// export the container_host env var and try again
+			cmd = exec.Command("export", "CONTAINER_HOST", fmt.Sprintf("ssh://%s@localhost", u.Username))
+			_, err = Start(cmd, GinkgoWriter, GinkgoWriter)
+			Expect(err).ToNot(HaveOccurred())
```

Comment:
export and unset are not commands, they are shell built-ins.
you need os.Setenv(0 and os .Unsetenv().
But I don't see how this tests anything `"--connection", "QA"` should overwrite the env var, no?